README.md

```diff
@@ -1,3 +1,87 @@
-# zs-rediz-rate-limiter
+# rediz-rate-limiter
 
-ZS Library: zs-rediz-rate-limiter
+Redis-based rate limiting.
+
+## Usage
+Rate-limiting is accomplished by creating an instance and calling the `#check`
+method, which will increment the ongoing count for the provided key or, if
+the limit has been reached, reject with a limit-exceeded error.
+
+Check calls with the same key will be tracked together, regardless of which
+process the call was made in, assuming the same redis database is used by
+each process.
+
+The check method requires a completion rate and burst count with each call.
+These can be changed between check calls to the same key, though in practice
+they usually should remain the same.
+
+```js
+const RateLimiter = require('rediz-rate-limiter');
+const RedizClient = require('rediz');
+const pasync = require('pasync');
+
+let client = new RedizClient({ /* redis config */});
+let limiter = new RateLimiter(client);
+
+
+// At most one per 2 seconds.
+let check = () => limiter.check('key', 0.5, 1);
+
+// First check will resolve.
+check()
+	// Second check will reject.
+	.then(() => check().catch(() => {}))
```

Comment:
I get that this is just example code, but this line will always pass no matter how `check` is implemented.  You could add a `then` stanza in which you throw an error if you want, which would essentially flip the success bit.